src/main/java/com/nike/cerberus/server/config/guice/CmsGuiceModule.java

```diff
@@ -164,68 +130,12 @@ private void configureAuditLogging() {
-            AuthenticateUser authenticateUser,
-            CodeHandlingMfaCheck mfaCheck,
-            RefreshUserToken refreshUserToken,
-            AuthenticateIamRole authenticateIamRole,
-            AuthenticateIamPrincipal authenticateIamPrincipal,
-            AuthenticateStsIdentity authenticateStsIdentity,
-            RevokeToken revokeToken,
-            GetAllRoles getAllRoles,
-            GetRole getRole,
-            GetSafeDepositBoxes getSafeDepositBoxes,
-            GetSafeDepositBoxV1 getSafeDepositBoxV1,
-            GetSafeDepositBoxV2 getSafeDepositBoxV2,
-            DeleteSafeDepositBox deleteSafeDepositBox,
-            UpdateSafeDepositBoxV1 updateSafeDepositBoxV1,
-            UpdateSafeDepositBoxV2 updateSafeDepositBoxV2,
-            CreateSafeDepositBoxV1 createSafeDepositBoxV1,
-            CreateSafeDepositBoxV2 createSafeDepositBoxV2,
-            GetSDBMetadata getSDBMetadata,
-            OverrideSdbOwner overrideSdbOwner,
-            PutSDBMetadata putSDBMetadata,
-            GetDashboardRedirect getDashboardRedirect,
-            GetDashboard getDashboard,
-            WriteSecureData writeSecureData,
-            ReadSecureData readSecureData,
-            DeleteSecureData deleteSecureData,
-            TriggerScheduledJob triggerScheduledJob,
-            RestoreSafeDepositBox restoreSafeDepositBox,
-            GetSecretVersionPathsForSdb getSecretVersionPathsForSdb,
-            GetSecureDataVersions getSecureDataVersions,
-            WriteSecureFile writeSecureFile,
-            ReadSecureFile readSecureFile,
-            HeadSecureFile headSecureFile,
-            GetSecureFiles getSecureFiles,
-            DeleteSecureFile deleteSecureFile,
-            GetAuthKmsKeyMetadata getAuthKmsKeyMetadata
-    ) {
-        return new LinkedHashSet<>(Arrays.<Endpoint<?>>asList(
-                healthCheckEndpoint,
-                robotsEndpoint,
-                // Cerberus endpoints
-                getAllCategories, getCategory, createCategory, deleteCategory,
-                authenticateUser, authenticateIamPrincipal, authenticateStsIdentity, mfaCheck, refreshUserToken,
-                authenticateIamRole, revokeToken,
-                getAllRoles, getRole,
-                getSafeDepositBoxes, getSafeDepositBoxV1, getSafeDepositBoxV2,
-                deleteSafeDepositBox, updateSafeDepositBoxV1, updateSafeDepositBoxV2, createSafeDepositBoxV1, createSafeDepositBoxV2,
-                getSDBMetadata, putSDBMetadata, overrideSdbOwner,
-                writeSecureData, readSecureData, deleteSecureData,
-                triggerScheduledJob,
-                getDashboard, getDashboardRedirect,
-                writeSecureFile, readSecureFile, deleteSecureFile, headSecureFile, getSecureFiles,
-                restoreSafeDepositBox,
-                getSecretVersionPathsForSdb, getSecureDataVersions, getAuthKmsKeyMetadata
-        ));
+    public Set<Endpoint<?>> appEndpoints(Injector injector) {
```

Comment:
Now we can in theory not have to manually add endpoints to this and add custom endpoints in an internal module.

For example Tim wants us to be able to map user groups to individuals, which is likely to be Nike specific.